examples/riot/buildAll.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+# List of folders
+FOLDERS=("blinky" "hello")
+
+# List of boards
+BOARDS=("native" "nucleo-f429zi")
```

Comment:
I allow to build multiple boards so that I can at least test `native` and an example board.
The `native` target runs `RIOT` on your linux system and usually can behave quite different to a concrete board.
So I think it is important to always also test for `native`.